client/src/data/copy/common.tsx

```diff
@@ -34,6 +41,13 @@ export const BETA_BANNER = defineMessages({
   },
 });
 
+export const TSD = defineMessages({
+  URL: {
+    id: 'common.pages.tsd.url',
+    defaultMessage: `https://static-data-screeningtool.geoplatform.gov/data-pipeline/data/score/downloadable/cejst_technical_support_document.pdf`,
```

Comment:
We should set the URL as i18n pattern, since this document will also come in Spanish

Reply:
@esfoobar-usds - Yes, this URL is already i18n'd. Are you implying something else? When we get the Spanish link it will be a new link and es.json will be :

```json
{
:
"common.pages.tsd.url" : "https://static-data-screeningtool.geoplatform.gov/data-pipeline/data/score/downloadable/cejst_technical_support_document_es.pdf",
:
}
```

Reply:
Oh! Great, then, it's ready!